host.json

```diff
@@ -0,0 +1,27 @@
+{
+        "name": "contivModel",
+        "objects": [
+                {
+                        "name": "Bgp",
```

Comment:
Can we call this 'host' object? 
We want to add more attributes to host object. 
It'll be convenient to add it to the same object instead of creating separate objects like host-Bgp and host-vxlan etc..